### PR TITLE
[shared] cleanup share versions

### PR DESCRIPTION
**Pull request by pppazos**
**Changelog**
- add default versions on shared roles and cleanup playbooks for:
    - **setup/kubectl** :  default in defaults/main.yaml to 1.16.3, can be override from playbook
    - **setup/helm**   default in defaults/main.yaml to v3.2.4, can be override from playbook
    - **setup/vault** default in defaults/main.yaml to v1.7.0, can be override from playbook
    - **setup/aws-auth** default in defaults/main.yaml to v1.10.3, can be override from playbook
    - **setup/ambassador** version is locked in helm chart

**Reviewed by**
@sownak , @suvajit-sarkar 

 

**Linked issue**
#1511

### CODE REVIEW

File: platforms/shared/configuration/roles/setup/aws-auth/defaults/main.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+tmp_directory: "{{ lookup('env', 'TMPDIR') | default('/tmp',true) }}"
+
+default:
+  version: "1.10.3"
+  release_date:  "2018-07-26"
+  checksum: ""
```
Comment (sownak) [inline platforms/shared/configuration/roles/setup/aws-auth/defaults/main.yaml:7]: newline

File: platforms/shared/configuration/roles/setup/helm/defaults/main.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+tmp_directory: "{{ lookup('env', 'TMPDIR') | default('/tmp',true) }}"
+
+default:
+  version: "v3.2.4"
+  checksum: ""
```
Comment (sownak) [inline platforms/shared/configuration/roles/setup/helm/defaults/main.yaml:6]: newline

File: platforms/shared/configuration/roles/setup/vault/defaults/main.yaml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+---
+tmp_directory: "{{ lookup('env', 'TMPDIR') | default('/tmp',true) }}"
+
+default:
+  version: "1.7.0"
+  checksum: ""
```
Comment (sownak) [inline platforms/shared/configuration/roles/setup/vault/defaults/main.yaml:6]: arent we using 1.7.1. also newline

**Review by sownak — CHANGES_REQUESTED** [review]

**Review by sownak — APPROVED** [review]

File: platforms/shared/configuration/roles/setup/vault/defaults/main.yaml (hunk 3 above)
Comment (pppazos) [reply platforms/shared/configuration/roles/setup/vault/defaults/main.yaml]: done.
upgraded vault client to 1.7.1 to match server running 1.7.1
enabled yamllint rule new-line-at-end-of-file in shared/configuration folder

**Pull request merged**